packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -24,11 +24,16 @@ export interface PluginConfig extends Partial<PluginWorkerConfig> {
 type Defined<T> = Exclude<T, undefined>;
 
 export type AssetsOnlyConfig = SanitizedWorkerConfig & {
+	topLevelName: Defined<SanitizedWorkerConfig['topLevelName']>;
+	name: Defined<SanitizedWorkerConfig['name']>;
+	compatibility_date: Defined<SanitizedWorkerConfig['compatibility_date']>;
 	assets: Defined<SanitizedWorkerConfig['assets']>;
 };
 
 export type WorkerConfig = SanitizedWorkerConfig & {
+	topLevelName: Defined<SanitizedWorkerConfig['topLevelName']>;
 	name: Defined<SanitizedWorkerConfig['name']>;
+	compatibility_date: Defined<SanitizedWorkerConfig['compatibility_date']>;
```

Comment:
removing the `compatibility_data` here and in `AssetsOnlyConfig` doesn't seem to break anything, is this really necessary?

Reply:
I think that as there is now an assertion for the `compatibility_date` it's good for the types to reflect that it will always be defined. Also means that you can't accidentally remove the assertion without a type error.

Reply:
and we want it to be there because it's needed in the file we generate on builds right?

ok makes sense to me, thanks 🙂